### PR TITLE
{AKS} get-versions using newer backend api

### PR DESCRIPTION
**Pull request by haitch**
**Related command**
az aks get-versions

**Description**
AKS offer a new simpler api for listKubernetesVersions, to replace legacy api ListOrchestratorProfile, which was designed to support docker swam and apache mesosphere. 

**Testing Guide**
compare out put between old and new az aks get-versions, data should remain same, but json/table output have minor changes.

**History Notes**
[aks] **BREAKING CHANGE**: Both json payload and table format changed for `az aks get-versions`

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Comment by FumingZhang** [comment]
Found some server-side issues, fixing

**Comment by zhoxing-ms** [comment]
> Found some server-side issues, fixing

Any update for the service issue? @haitch @FumingZhang

**Comment by zhoxing-ms** [comment]
Could you please resolve these CI issues?

**Comment by zhoxing-ms** [comment]
@haitch Please note that we are launching the release for this sprint this week. Please resolve all comments by tomorrow, otherwise the release of this PR will have to be postponed to the next sprint (on **07-04**)

File: src/azure-cli/azure/cli/command_modules/acs/commands.py [hunk 1]
```diff
@@ -27,14 +27,6 @@
 # pylint: disable=too-many-statements
```
Comment (FumingZhang) [inline src/azure-cli/azure/cli/command_modules/acs/commands.py:27]: As taking thew new API from the managed cluster operation group, the previous container service client factory (on line 6) should be removed.

> src/azure-cli/azure/cli/command_modules/acs/commands.py:6:0: W0611: Unused cf_container_services imported from azure.cli.command_modules.acs._client_factory (unused-import)

**Pull request closed without merging**